src/main/java/com/revature/controller/JobController.java

```diff
@@ -73,8 +76,41 @@ public JobController(JobService jobService, SkillService skillService) {
 	 the page to search for jobs.
 	 * @param page An integer identifying the page to search for jobs.
 	 * @return A list of all jobs on the specified page. */
-    public List<Job> getAllPaged(@PathVariable int page){
-    	return jobService.getAllJobsPaged(page);
+    public List<Job> getAllPaged(@PathVariable int page,
+			@RequestParam("filtervalue") String value, @RequestParam("skillids") String data) {
+    	boolean useFilter = false;
+    	if (!value.isEmpty() || !data.isEmpty()) {
+    		useFilter = true;
+    	}
+    	if (useFilter) {
```

Comment:
useFilter should be removed. This if statement should have the same check that the one on line 82 has.

Reply:
And actually, you can use DeMorgan's rule to simplify the if statement.

Reply:
fixed